tcms/testcases/tests/test_views.py

```diff
@@ -564,3 +566,33 @@ def test_disabled_priority_now_shown(self):
         self.assertEqual(HTTPStatus.OK, response.status_code)
         self.assertContains(response, 'Set P3')
         self.assertNotContains(response, 'Set P4')
+
+
+class TestShowCases(BasePlanCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.request = RequestFactory()
```

Comment:
You don't need a class attribute here. In every test you can create the request variable at the beginning of the test. You don't gain anything by having a setup in this scenario.